custom-words.txt

```diff
@@ -2752,4 +2752,4 @@ remediatable
 Overprovisioning
 Unprepares
 Unpreparing
-
+wasbs
```

Comment:
What is this? Do we need it?

Reply:
it's an acronym for Windows Azure Storage BLob